File: help/eng_gb/exercicios_da_ficha_3/Ficha03_005/Ficha03_005.cpp
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

#include <iostream>
#include <cstring>
#include <cmath>
using namespace std;

// Class definition
class Ficha03_005 {

    public:
        int main() {
            cout << "Pythagorean Theorem\n";
            cout << "Catet a \t: ";
            float a;
            cin >> a;
            cout << "Catet b \t: ";
            float b;
            cin >> b;
            float c  = sqrt( a * a + b * b ) ;
            cout << "Hypotnusa \t: " << c ;
            return 0;
        }
};// end of class Ficha03_005
// Execute main function
int main(){
    Ficha03_005 application; // create object
    application.main(); // execute object
    return 0;
}

//C++  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Tue, 2022 Sep 13 21:55

File: help/eng_gb/exercicios_da_ficha_3/Ficha03_005/Ficha03_005.java
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

import java.util.Scanner;

public class Ficha03_005 {

    static Scanner keyb = new Scanner(System.in);//keyboard

    //Main Function
    public static void main( String[] args) {
        System.out.print("Pythagorean Theorem\n" );
        System.out.print("Catet a \t: " );
        double a = Double.valueOf(keyb.nextLine());
        System.out.print("Catet b \t: " );
        double b = Double.valueOf(keyb.nextLine());
        double c = Math.sqrt( a * a + b * b ) ;
        System.out.print("Hypotnusa \t: " + c );
    }

}
//Java  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Tue, 2022 Sep 13 21:55
